go/private/rules/transition.bzl

```diff
@@ -109,6 +110,10 @@ def _go_transition_impl(settings, attr):
     if tags:
         settings["//go/config:tags"] = _deduped_and_sorted(tags)
 
+    gofips140 = getattr(attr, "gofips140", "off")
```

Comment:
Doesn't this mean that it's impossible to disable fips for a binary if it is enabled globally?